app/src/main/java/com/infomaniak/mail/utils/ErrorCode.kt

```diff
@@ -36,6 +36,7 @@ object ErrorCode {
 
     //region Mailbox
     const val MAILBOX_LOCKED = "mailbox_locked"
+    const val ACCESS_DENIED = "access_denied"
```

Comment:
I'd like to wait a week or two before merging this.

I just merged a fix that should reduce the number of `access_denied` we receive.
I'd like to see the result of this fix before implementing this feature.

Reply:
I was able to check Sentry, so we can merge now !

Thx.